### ISSUE TITLE
Reader Models & Training Setup/HPMs for RAG

### ISSUE THREAD

**Issue by drothermel** [samsam3232/qampari#3]
Hey, thanks for the dataset & code!  Do you have any intention to release the final reader model(s)?  Specifically I'm interested in the RAG models.  If not, do you have additional details on your training setup, like learning rate (and accompanying effective batch size), total training steps, initial model name (t5-base?), etc used to produce the results in the README? Thanks!

**Comment by samsam3232** [comment]
Hi,
We will upload the models + a new version of the paper on Sunday next week

**Comment by samsam3232** [comment]
Hi, sorry for the delay, we added the models and updated the papers.
Good luck !
